test/unit/mdc-select/mdc-select-enhanced.test.js

```diff
@@ -33,7 +33,7 @@ import {MDCSelect} from '../../../packages/mdc-select/index';
 import {cssClasses, strings} from '../../../packages/mdc-select/constants';
 import {MDCNotchedOutline} from '../../../packages/mdc-notched-outline/index';
 import {MDCMenu, MDCMenuFoundation} from '../../../packages/mdc-menu/index';
-import {MDCMenuSurfaceFoundation} from '../../../packages/mdc-menu-surface/index';
+import {MDCMenuSurfaceFoundation} from '../../../packages/mdc-menu-surface';
```

Comment:
revert this change. It probably needs to be `index.ts`

Reply:
The tests pass as-is, so why does it matter?